roles/oidc-playground-server/handlers/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: systemctl daemon-reload
+  systemd:
+    daemon_reload: yes
+
+- name: restart oidc-playground
+  service:
+    name: oidc-playground
+    state: restarted
+
+- name: update_ca_trust
+  command: update-ca-trust
```

Comment:
I don't see this handler used in this role, so it can be removed here

Reply:
Done